LICENSE

```diff
@@ -0,0 +1,11 @@
+Copyright 2017 Open Observatory of Network Interference (OONI), The Tor Project
```

Comment:
The date should be bumped to 2019. I will do that in https://github.com/ooni/license too.